src/containers/ProjectContentsContainer.jsx

```diff
@@ -9,6 +9,16 @@ import * as contentsActions from '../ducks/resource';
 import isElementTranslatable from '../helpers/isElementTranslatable';
 
 class ProjectContentsContainer extends Component {
+  static getTextFromPath(resource, path) {
```

Comment:
I don't think it's used anywhere except in this class, so it makes more sense, to me, to define it here. I'm not massively fussed either, but eslint was warning about class methods that don't act on an instance of the class.